client/lib/billing-card/app/serializers/billing-card.js

```diff
@@ -0,0 +1,2 @@
+import TaskSerializer from 'tahi/serializers/task';
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'TaskSerializer' is not defined.<br>'from' is not defined.